docs/docs/guide/05-nameservice/00-tutorial.md

```diff
@@ -17,8 +17,8 @@ in the Ignite CLI Developer Tutorials. Before you start this building your
 nameservice app, we recommend that you complete these foundational tutorials:
 
 - [Install Ignite CLI](../01-install.md)
-- [Hello, World](../03-hello.md)
-- [Module Basics](../04-blog/00-build-blog.md)
+- [Hello, World](/guide/hello)
+- [Module Basics](/guide/blog)
```

Comment:
Those links point to the stable version, as mentioned in the docusaurus documentation [0], you need to prefix the link with `./` to keep the current version (nightly here). 

This is the fixed version : 

```suggestion
- [Hello, World](./hello/express)
- [Module Basics](./blog/express)
```

They are a little strange, at first I wanted to write then `./guide/hello` and `./guide/blog` but that wasn't working. Seems like docusaurus is doing some black magic with the links.

You can remove the `express` suffix if you set the `slug` yaml parameter on top of each `00-express.md`. Respectively it should be set to `slug: /guide/hello` and `slug: /guide/blog` for the hello and blog tutorial.

Once `slug` is set, the links can be like : 

```
- [Hello, World](./hello)
- [Module Basics](./blog)
```

[0] https://docusaurus.io/docs/versioning#terminology

Reply:
@tbruyelle I can't find Docusaurus documentation explaining the `/express` suffix and the `00-express.md`. Where does that behaviour comes from?

Reply:
I have no idea about the documentation but this is what appears in the address bar when you go manually to the hello or blog express tutorial. Seems like somehow it's able to interpret `express` as `00-express.md`. 

What's *funny* is it cannot do the inverse, if you write the link `./hello/00-express.md` you get a 404...